jekyll/_cci2/credits.md

```diff
@@ -1,55 +0,0 @@
----
```

Comment:
RE the files you have converted, dynamic config and the markdown FAQ are moved to /archived, but credits and scheduled pipelines are just deleted. As the pages do still exist I think it's fine to just delete rather than archive the markdown?